specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/preview/2018-09-01/Backups.json

```diff
@@ -213,6 +213,11 @@
           "type": "string",
           "readOnly": true
         },
+        "isCloudRecoveryReady": {
```

Comment:
Pretty much this means the back is completed?

Reply:
Actually, what is the purpose of this field?  You already have a status field.

Reply:
add a new property to exisiting release is treated as breaking change, pls do it in a new release.

Reply:
Originally, all but two roles from a full backup are required for cloud recovery, so a simple backup succeeded/failed status was good enough to indicate if the backup is good for cloud recovery.

Now we have onboarded more infra service backups for services healing. These backups are taken in case any infra service needs repairing to avoid a cloud recovery, and most of them are not required for cloud recovery.

The ‘cloud recovery essential’ backups and the ‘nice to have’ backups are taken at the same time when an infra backup is triggered. If some ‘nice to have’ backup snapshots failed, the backup can still be used for cloud recovery. The new property together with the original full backup status help the customers to understand their situation better and react accordingly.

Reply:
> Actually, what is the purpose of this field? You already have a status field.

Originally, all but two roles from a full backup are required for cloud recovery, so a simple backup succeeded/failed status was good enough to indicate if the backup is good for cloud recovery.

Now we have onboarded more infra service backups for services healing. These backups are taken in case any infra service needs repairing to avoid a cloud recovery, and most of them are not required for cloud recovery.

The ‘cloud recovery essential’ backups and the ‘nice to have’ backups are taken at the same time when an infra backup is triggered. If some ‘nice to have’ backup snapshots failed, the backup can still be used for cloud recovery. The new property together with the original full backup status help the customers to understand their situation better and react accordingly.

Reply:
> add a new property to exisiting release is treated as breaking change, pls do it in a new release. definition of breaking change https://github.com/Azure/adx-documentation-pr/wiki/Breaking-changes-guidelines

There is no PUT api for that BackupInfo, thus no GET-PUT pipeline, so it can be regarded as non-breaking change, is that acceptable?

Reply:
> > add a new property to exisiting release is treated as breaking change, pls do it in a new release. definition of breaking change https://github.com/Azure/adx-documentation-pr/wiki/Breaking-changes-guidelines
> 
> There is no PUT api for that BackupInfo, thus no GET-PUT pipeline, so it can be regarded as non-breaking change, is that acceptable?

make sense.